middlewares/verifyToken.js

```diff
@@ -0,0 +1,27 @@
+import configs from 'configs/index';
+import jwt, { TokenExpiredError } from 'jsonwebtoken';
+import { UnauthorizedError, InvalidTokenError } from 'common/error';
+
+const secretKey = configs.SECRET_KEY;
+
+function verifyToken(req, res, next) {
+  const bearerHeader = req.headers.authorization;
+  if (bearerHeader) {
+    const splitedHeader = bearerHeader.split(' ');
+    const token = splitedHeader[1];
+    jwt.verify(token, secretKey, (err, result) => {
+      if (err) {
+        if (err instanceof TokenExpiredError) {
+          throw new InvalidTokenError({ code: 40302, message: 'TokenExpired' });
```

Comment:
Use 2 different type of error class for 2 error

Reply:
Done

Reply:
https://github.com/EasyEventV2/ee2_api/pull/1/commits/2f849054ef50719c8e970815dd43b0004ed521d4